### DISCUSSION TITLE
HomeKit Rolladenstand unterscheidet sich in KNX

### DISCUSSION THREAD

**Discussion by Varone3000** [brutella/hkknx-public#280] [Q&A] [answered]
Hallo zusammen,

ich habe folgendes Problem die % Zahlen unterscheiden sich zwischen Hkknx und KNX. 

In KNX ist ja 100% geschlossen und 0% offen. 

In HKKNX bzw Homekit genau anders rum. 100% ist Offen. 
zumindestens wenn ich Befehle mit Siri gebe. 

Hab ich einen Konfigurationsfehler?

**Comment by brutella** [comment, marked as answer]
Das ist so bei HomeKit und kann nicht geändert werden. 🤷‍♂️

**Reply by Varone3000** [reply]
Danke für die schnelle Antwort.

Hast du die Möglichkeit bei Apple einen Feature Request zu machen? Gruß

**Reply by brutella** [reply]
Ich bin mir ziemlich sicher, dass Apple so etwas nicht ändern wird.
